testing-overhead/src/test/java/io/opentelemetry/config/TestConfig.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package io.opentelemetry.config;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Defines a test config.
+ */
+public class TestConfig {
+
+  private final static int DEFAULT_MAX_REQUEST_RATE = 0;  // none
+  private final static int DEFAULT_CONCURRENT_CONNECTIONS = 5;
+  private final static int DEFAULT_TOTAL_ITERATIONS = 500;
```

Comment:
500 iterations seems too little. Ideally there should be a warmup iteration before measurement iteration so that we would measure application in steady state where it has already jit compiled everything it can.

Reply:
That's reasonable -- I think I agree that 500 is too little and we should probably bump it up once things settle down and we're actually persisting data. For development, 500 is nice because you don't have to wait _too_ long.

As far as warmup -- I'm torn, because if the agent impacts startup and jit (with more code, the jitter will probably have to work harder, right?) -- maybe we _do_ want to account for that?  Does allowing the app to reach steady state seem slightly unfair? Maybe we're also talking about two different types of tests, so I could see this maybe becoming a parameter of the configuration.  🤔

Reply:
> Maybe we're also talking about two different types of tests

👍

Reply:
Our users are likely to care less about startup overhead than steady state performance.

Reply:
Ok. @laurit would you be ok with deferring a warmup enhancement for a future PR?